cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -41,12 +41,13 @@ function LogViewerController ($scope, $window, LogViewerStore, myLogsApi, LOGVIE
   vm.setProgramMetadata = (status) => {
     vm.programStatus = status;
 
-    if(!vm.entityName) {
+    if (!vm.entityName) {
       vm.entityName = vm.programId;
     }
 
-    switch(status){
+    switch (status) {
       case 'RUNNING':
+      case 'STARTING':
```

Comment:
we should get a list from someone in the backend about the comprehensive list of statuses, just to make sure